js/components/plates.js

```diff
@@ -109,6 +109,7 @@ export default class Plates extends PureComponent {
     this.restoreSnapshot = this.restoreSnapshot.bind(this)
     this.restoreInitialSnapshot = this.restoreInitialSnapshot.bind(this)
     this.handleResize = this.handleResize.bind(this)
+    this.isStateObjectEmpty = this.isStateObjectEmpty.bind(this)
```

Comment:
Not necessary unless you pass this function via reference somewhere (e.g. as a property or event handler).